src/main/java/ChatReques6.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
Using wildcard types like List<?> can lead to type safety issues. It's better to specify the type explicitly.